pulumi/infra/__init__.py

```
@@ -0,0 +1 @@
+from . import logging_bucket, pulumi_state  # noqa: F401
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)